ui/raidboss/data/07-dt/trial/valigarmanda-ex.txt

```diff
@@ -0,0 +1,171 @@
+# -r MjxcGWaA73mF8B1J -rf=104
+# -ii 8FF3 8FF5 8FF6 8FF7 8FF8 8FF9 8FE4 8FE6 8FCB 8FFA 9013 9014 9015 901A 900D 900E 900F 9012 9016 8FC7 8FD9 87A2 9002 9003 9006 6854 9566 9567 95B3 8FF4 8FDE 8FDD 8FDF 8FD7 8FD5
+# -p 9008:1006.7 95C3:2005.7 8FD1:3005.7 9007:4003.7
+# -it "Valigarmanda" "Flame-kissed Beacon"
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+6.0 "--sync--" Ability { id: "8FF1", source: "Valigarmanda" }
+18.0 "Spikesicle" Ability { id: "8FF2|8FF4", source: "Valigarmanda" } duration 11.8
+# Too spammy to display each of these, and too close together to sync properly.
+26.0 "Sphere Shatter x10" #Ability { id: "995D", source: "Ice Boulder" } duration 10.8
+
+# Fire mini-phase
+# Skyruin has two separate casts, one for boss animation and one for actual damage.
+# The animation cast time ends while the on-screen cast bar is still running,
+# so sync to but don't display the first ability
+40.7 "--sync--" Ability { id: "95C4", source: "Valigarmanda" }
+46.1 "Skyruin (fire phase)" Ability { id: "8FD4", source: "Valigarmanda" }
+54.8 "Triscourge" Ability { id: "8FE7", source: "Valigarmanda" }
+62.9 "Volcanic Drop" Ability { id: "8FE5", source: "Valigarmanda" } duration 4.5
+# Strangling Coil, Slithering Strike, or Susurrant Breath
+76.3 "Middle/Away/Front Corners" Ability { id: "8FC8|8FCC|8FD0", source: "Valigarmanda" }
```

Comment:
For “middle/away/front corners”, could we change it to include all three actual ability names in the timeline file, and then label it as you have it here via translation replacement? Or is that going to be absolutely too long even for a timeline file?